sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsError.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// </summary>
+    public readonly struct TextAnalyticsError
```

Comment:
So this model doesn't have the InnerError anymore?

Reply:
That's correct.